addons/base_address_extended/models/res_country.py

```diff
@@ -4,15 +4,10 @@
 from odoo import fields, models
 
 
-class ResCountry(models.Model):
+class Country(models.Model):
     _inherit = 'res.country'
 
-    street_format = fields.Text(
-        help="Format to use for streets belonging to this country.\n\n"
-             "You can use the python-style string pattern with all the fields of the street "
-             "(for example, use '%(street_name)s, %(street_number)s' if you want to display "
-             "the street name, followed by a comma and the house number)"
-             "\n%(street_name)s: the name of the street"
-             "\n%(street_number)s: the house number"
-             "\n%(street_number2)s: the door number",
-        default='%(street_number)s/%(street_number2)s %(street_name)s', required=True)
+    enforce_cities = fields.Boolean(
+        string='Enforce Cities',
+        help="Check this box to ensure every address created in that country has a 'City' chosen "
+             "in the list of the country's cities.")
```

Comment:
Do we still need this checkbox? We could add `city_id` into the address view using the custom `ir.ui.view` when needed

Reply:
Indeed, with base_address_extended we update the default form view to use city_id if enforce_countries. This allows to enable city_id for any country